ykrt/src/mt.rs

```diff
@@ -557,6 +569,14 @@ impl MT {
             }
         }
     }
+
+    #[cfg(yk_llvm_sync_hack)]
+    pub fn llvm_sync_hack(&self) {
```

Comment:
I should be violently objecting to a spinlock, but since this really isn't designed to be permanent, I guess it's ok for now. Agree?

Reply:
It's terrible but, as you say, temporary. Anything more sophisticated won't (I hope!) be worth the effort.